test/common/globals.js

```diff
@@ -63,6 +63,7 @@ const intrinsics = new Set([
   'SharedArrayBuffer',
   'Atomics',
   'WebAssembly',
+  'Iterator',
```

Comment:
We probably also want an entry in the `.eslintrc.yaml` like https://github.com/nodejs/node/blob/147abb99d1478b3d88be38f0c9c0ad9ed6a7af73/lib/.eslintrc.yaml#L66-L67